spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -883,14 +883,14 @@ def mock_column(name, type, options = {})
     end
   end
 
-  describe '#files_by_pattern' do
+  describe '.files_by_pattern' do
     subject { AnnotateModels.files_by_pattern(root_directory, pattern_type, options) }
 
-    context 'when pattern_type=additional_file_patterns' do
-      let(:pattern_type) { 'additional_file_patterns' }
+    context 'when pattern_type is "additional_file_patterns"' do
       let(:root_directory) { nil }
+      let(:pattern_type) { 'additional_file_patterns' }
 
-      context 'with additional_file_patterns' do
+      context 'when additional_file_patterns is specified is the options' do
```

Comment:
Did you mean to put `... is specified in the options`?

Reply:
Yes, I mean to put is specified in the options...